src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -787,6 +787,7 @@ def test_backup_job(self, resource_group, vault_name, vm_name, storage_account):
 
         self.cmd('backup job stop -g {rg} -v {vault} -n {job}')
 
+    @unittest.skip('CloudInternalError: Microsoft Azure Backup encountered an internal error. Wait for a few minutes and then try the operation again. If the issue persists, please contact Microsoft support.')
     @ResourceGroupPreparer()
```

Comment:
@zubairabid At present, this test cannot be passed in live mode. To avoid blocking this PR merge, we skip this test first. Could you help to submit a separate new PR to fix the live test?